src/msg/async/Stack.cc

```diff
@@ -84,28 +84,28 @@ std::shared_ptr<NetworkStack> NetworkStack::create(CephContext *c, const string
 
 Worker* NetworkStack::create_worker(CephContext *c, const string &type, unsigned worker_id)
 {
+  Worker* worker = nullptr;
   if (type == "posix")
-    return new PosixWorker(c, worker_id);
+    worker = new PosixWorker(c, worker_id);
 #ifdef HAVE_RDMA
   else if (type == "rdma")
-    return new RDMAWorker(c, worker_id);
+    worker = new RDMAWorker(c, worker_id);
 #endif
 #ifdef HAVE_DPDK
   else if (type == "dpdk")
-    return new DPDKWorker(c, worker_id);
+    worker = new DPDKWorker(c, worker_id);
 #endif
 
-  lderr(c) << __func__ << " ms_async_transport_type " << type <<
-    " is not supported! " << dendl;
-  ceph_abort();
-  return nullptr;
+  const int InitEventNumber = 5000;
+  worker->center.init(InitEventNumber, worker_id, type);
```

Comment:
If worker == NULL you will crash on this line before ceph_assert().

Reply:
Done. Wait for updating PR.